recipes/installer/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "installer" %}
```

Comment:
I wonder if we should namespace this one and call it `python-installer` :-/

Reply:
No hard opinion on that point so let me know what you prefer or we can wait for more inputs.

Reply:
@ocefpaf What about `pdm-installer` to be more specific? Not sure what is easiest.

Reply:
If that makes sense, sure. My only concern is with "installer" alone. Too generic.

Reply:
`installer` is not specific to `pdm` (unless I miss something here) so I don't think it should be namespace with `pdm-`. `python-installer` seems more generic here.

Reply:
I have renamed it to `python-installer`.